metrics_utility/test/ccspv_reports/test_CCSPv2_with_no_data.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from conftest import validate_sheet_columns, validate_sheet_tab_names
+
+from metrics_utility.test.util import run_build_ext
+
+
+env_vars = {
+    'METRICS_UTILITY_PRICE_PER_NODE': '11.55',
+    'METRICS_UTILITY_REPORT_RHN_LOGIN': 'test_login',
+    'METRICS_UTILITY_SHIP_PATH': './metrics_utility/test/test_data',
+    'METRICS_UTILITY_REPORT_SKU_DESCRIPTION': 'EX: Red Hat Ansible Automation Platform, Full Support (1 Managed Node, Dedicated, Monthly)',
+    'METRICS_UTILITY_REPORT_H1_HEADING': 'CCSP NA Direct Reporting Template',
+    'METRICS_UTILITY_REPORT_PO_NUMBER': '123',
+    'METRICS_UTILITY_SHIP_TARGET': 'directory',
+    'METRICS_UTILITY_REPORT_COMPANY_NAME': 'Partner A',
+    'METRICS_UTILITY_REPORT_SKU': 'MCT3752MO',
+    'METRICS_UTILITY_REPORT_EMAIL': 'email@email.com',
+    'METRICS_UTILITY_REPORT_TYPE': 'CCSPv2',
+}
+
+file_path = './metrics_utility/test/test_data/reports/2024/02/CCSPv2-2024-01.xlsx'
```

Comment:
```suggestion
file_path = './metrics_utility/test/test_data/reports/2024/01/CCSPv2-2024-01.xlsx'
```

mismatch between the month in the filename & the month dir